installation-and-configuration/configure-winrm-authentication.md

```diff
@@ -0,0 +1,29 @@
+---
+description: >-
+  This article explains how to configure WinRm authentication on
+  your machine to successfully run snapshots.
+---
+
+# Configure Windows Remote Management (WinRM)
+
+In order to successfully collect data, **SysKit Trace** will create a **remote PowerShell session** to the Security & Compliance Center.
```

Comment:
maybe mention all of the workloads that use remote powershell? or phrase it differently

Reply:
Added workloads affected